src/service.ts

```diff
@@ -58,7 +63,7 @@ export class PisaService extends StartStopService {
         this.applyMiddlewares(app, config);
 
         // block cache and processor
-        const cacheLimit = config.maximumReorgLimit === undefined ? 200 : config.maximumReorgLimit;
+        const cacheLimit = !config.maximumReorgLimit ? 200 : config.maximumReorgLimit;
```

Comment:
In general, for "primitive or null" and "primitive or undefined" we should prefer comparing with "== null" or "== undefined", as the comparison would fail if config.maximumReorgLimit is 0 (though not a valid choice in this case).

Reply:
Yeah, I agree, 0 isnt a valid choice - but I'll check that somewhere else instead.